lang/bosniaandherzegovina/en/holidays.json

```diff
@@ -0,0 +1,25 @@
+{
+    "Nova godina - prvi dan": "New Year - first day",
```

Comment:
These files about bosnia & ecuador shouldn't be in this PR.

Reply:
Above file removed

Reply:
I can not see any language folder here - https://github.com/spatie/holidays/pull/220/files
Let us know which files you want us to remove.

Reply:
@Nielsvanpach All extra files are removed. Please check and Let us know.